taxi/views.py

```diff
@@ -87,3 +90,31 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreateForm
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+def car_driver_update(request: HttpRequest, pk: int) -> HttpResponse:
+    user = request.user
+    car = Car.objects.get(pk=pk)
+    if user in car.drivers.all():
+        car.drivers.remove(user)
+    elif user not in car.drivers.all():
```

Comment:
Because of this `elif` - you have additional request to your `DB`. Use `else`, because there can be only 2 cases:
1. User will be inside of db and assigned to this car
2. User wasn't assigned to this car